.github/workflows/release_version.yml

```diff
@@ -0,0 +1,21 @@
+name: releaseVersion
```

Comment:
Very nice! Should this not be in the Gradle plugin instead though?

Reply:
I don't think so, this is not a plugin, it's just a lib. And libs live in the Maven Central.

Reply:
Right, I was tripped up by the `ORG_GRADLE_PROJECT_signingKey:` names - never mind then 👍